src/components/application_manager/src/commands/mobile/system_request.cc

```diff
@@ -54,6 +57,302 @@ uint32_t SystemRequest::index = 0;
 const std::string kSYNC = "SYNC";
 const std::string kIVSU = "IVSU";
 
+
+#define QUERY_APPS_VALIDATION_FAILED ":QUERY_APPS_VALIDATION_FAILED:"
+
+
+
+struct comparer
```

Comment:
@okoreniak Type names start with a capital letter and have a capital letter for each new word, with no underscores: MyExcitingClass, MyExcitingEnum.